ee/control/control.go

```diff
@@ -366,20 +364,30 @@ func (cs *ControlService) fetchAndUpdate(subsystem, hash string) error {
 	return nil
 }
 
+// knownSubsystem checks our registered consumers and subscribers to see if the given
+// subsystem is one that has been registered with the control service.
+func (cs *ControlService) knownSubsystem(subsystem string) bool {
+	if _, subsystemFromConsumer := cs.consumers[subsystem]; !subsystemFromConsumer {
+		if _, subsystemFromSubscriber := cs.subscribers[subsystem]; !subsystemFromSubscriber {
+			return false
+		}
+	}
```

Comment:
```suggestion
	if _, subsystemFromConsumer := cs.consumers[subsystem]; subsystemFromConsumer {
	  return true
	}
	if _, subsystemFromSubscriber := cs.subscribers[subsystem]; subsystemFromSubscriber {
			return true
	}
	
	return false
```

I think? But it seems fine either way

Reply:
Oh yup, reads better without the additional nesting -- updating now